indykite/resource_ingest_pipeline.go

```diff
@@ -109,7 +110,6 @@ func resourceIngestPipelineFlatten(
 ) diag.Diagnostics {
 	var d diag.Diagnostics
 	ipCfg := resp.GetConfigNode().GetIngestPipelineConfig()
-	setData(&d, data, ingestPipelineAppAgentTokenTypeKey, ipCfg.GetAppAgentToken())
```

Comment:
Add test for this. Meaning return token in format of `*****`

Reply:
Fixed